common/alert/reports/a11y-1.json

```
@@ -0,0 +1,908 @@
+        {
+            "earl:test": "WCAG2xTech:general/G207",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        }
+      ]
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.4.12",
+      "earl:test": "WCAG21:text-spacing",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "added class='test-textSpacing' to check text is not clipped",
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:css/C36",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        },
+        {
+            "earl:test": "WCAG2xTech:css/C35",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        },
+        {
+            "earl:test": "WCAG2xTech:failures/F104",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        },
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_1.4.13",
+      "earl:test": "WCAG21:content-on-hover-or-focus",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:inapplicable",
+      "dct:description": "",
+      "earl:result": [],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_2.1.1",
+      "earl:test": "WCAG21:keyboard",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "Used tab & shift-tab to check navigation on page",
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:general/G202",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        },
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_2.1.2",
+      "earl:test": "WCAG21:no-keyboard-trap",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:passed",
+      "dct:description": "Use tab key to check if you're stuck on component; Often use ESC key to come out of component",
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:general/G21",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        },
+        {
+            "earl:test": "WCAG2xTech:failures/F10",
+            "earl:outcome": "earl:passed",
+            "earl:subject": "_:subject"
+        }
+      ]
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "@id": "_:sc_2.1.4",
+      "earl:test": "WCAG21:character-key-shortcuts",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:inapplicable",
+      "dct:description": "checked for 'accesskey' attribute, which is not present. ALso checked for JS for keyup & keydown events"",
```

To fix the CI errors about JSON validation
```suggestion
      "dct:description": "checked for 'accesskey' attribute, which is not present. ALso checked for JS for keyup & keydown events",
```